tests/core/test_state_sync.py

```diff
@@ -2225,12 +2225,12 @@ def test_snapshot_batching(state_sync, mocker, make_snapshot):
         call(
             exp.to_table("sqlmesh._snapshots"),
             where=parse_one(
-                f"(name, identifier) in (('\"a\"', '{snapshot_b.identifier}'), ('\"a\"', '{snapshot_c.identifier}'))"
+                f"(name, identifier) in (('\"a\"', '{snapshot_b.identifier}'), ('\"a\"', '{snapshot_a.identifier}'))"
```

Comment:
Why did this test change?

Reply:
Because of the new hash values, so now the hash of Snapshot A is greater than that of Snapshot C and so in the first batch when they're ordered it is going to be A instead.